app/client/src/modules/ui-builder/ui/wds/WDSCustomWidget/component/customWidgetscript.js

```
@@ -0,0 +1,332 @@
+  CUSTOM_WIDGET_UPDATE_HEIGHT: "CUSTOM_WIDGET_UPDATE_HEIGHT",
+};
+
+// Function to create a communication channel to the parent
+export const createChannelToParent = () => {
+  const onMessageMap = new Map();
+
+  // Function to register an event handler for a message type
+  function onMessage(type, fn) {
+    let eventHandlerList = onMessageMap.get(type);
+
+    if (eventHandlerList && eventHandlerList instanceof Array) {
```

Biome (1.9.4)

[error] Use Array.isArray() instead of instanceof Array.

instanceof Array returns false for array-like objects and arrays from other execution contexts.
Unsafe fix: Use Array.isArray() instead.


(lint/suspicious/useIsArray)